components/cli/pkg/commands/build.go

```diff
@@ -19,323 +19,130 @@
-	}
-	execError := ""
-	stderrReader, _ := cmd.StderrPipe()
-	stderrScanner := bufio.NewScanner(stderrReader)
-	go func() {
-		for stderrScanner.Scan() {
-			execError += stderrScanner.Text()
-		}
-	}()
-	var stdout, stderr bytes.Buffer
-	cmd.Stdout = &stdout
-	cmd.Stderr = &stderr
-	err = cmd.Start()
-	if err != nil {
-		spinner.Stop(false)
-		errStr := string(stderr.Bytes())
-		fmt.Printf("%s\n", errStr)
-		util.ExitWithErrorMessage("Error occurred while building image", err)
-	}
-	err = cmd.Wait()
-	if err != nil {
-		spinner.Stop(false)
-		fmt.Println()
-		fmt.Printf("\x1b[31;1m\nBuild Failed.\x1b[0m %v \n", execError)
-		fmt.Println("\x1b[31;1m======================\x1b[0m")
-		errStr := string(stderr.Bytes())
-		fmt.Printf("\x1b[31;1m%s\x1b[0m", errStr)
-		util.ExitWithErrorMessage("Error occurred while building image", err)
-	}
-	_ = os.Remove(tempBuildFileName)
-	outStr := string(stdout.Bytes())
-	fmt.Printf("\r\x1b[2K\033[36m%s\033[m\n", outStr)
-
-	generateMetaData(parsedCellImage, targetDir, spinner)
-
-	folderCopyError := util.CopyDir(targetDir, filepath.Join(projectDir, constants.ZIP_ARTIFACTS))
-	if folderCopyError != nil {
-		spinner.Stop(false)
-		util.ExitWithErrorMessage("Error occurred creating image", err)
-	}
-	err = util.CleanOrCreateDir(filepath.Join(projectDir, constants.ZIP_BALLERINA_SOURCE))
-	if err != nil {
-		spinner.Stop(false)
-		util.ExitWithErrorMessage("Error occurred while creating the image", err)
-	}
-	fileCopyError := util.CopyFile(fileName, filepath.Join(projectDir, constants.ZIP_BALLERINA_SOURCE, filepath.Base(fileName)))
-	if fileCopyError != nil {
-		spinner.Stop(false)
-		util.ExitWithErrorMessage("Error occurred creating cell image", err)
-	}
-	balTomlParent := filepath.Dir(filepath.Dir(filepath.Join(currentDir, fileName)))
-	balTomlfileExist, err := util.FileExists(filepath.Join(balTomlParent, constants.BALLERINA_TOML))
-
-	if err != nil {
-		util.ExitWithErrorMessage("Error occurred while checking if Ballerina.toml exists", err)
-	}
-	if balTomlfileExist {
-		fileCopyError = util.CopyFile(filepath.Join(balTomlParent, constants.BALLERINA_TOML),
-			filepath.Join(projectDir, constants.ZIP_BALLERINA_SOURCE, constants.BALLERINA_TOML))
-		if fileCopyError != nil {
-			util.ExitWithErrorMessage(fmt.Sprintf("error occured while copying the %s", constants.BALLERINA_TOML), fileCopyError)
-		}
-		fileCopyError = util.CopyDir(filepath.Join(balTomlParent, constants.BALLERINA_LOCAL_REPO),
-			filepath.Join(projectDir, constants.ZIP_BALLERINA_SOURCE, constants.BALLERINA_LOCAL_REPO))
-	}
-	isTestDirExists, _ := util.FileExists(constants.ZIP_TESTS)
-	folders := []string{constants.ZIP_ARTIFACTS, constants.ZIP_BALLERINA_SOURCE}
-
-	if isTestDirExists {
-		folders = append(folders, constants.ZIP_TESTS)
+	var fileExist bool
+	if fileExist, err = util.FileExists(fileName); err != nil {
```

Comment:
ideally we should not have this util pkg.. lets handle it later.

Reply:
Ack.